dot/telemetry/telemetry.go

```diff
@@ -122,33 +94,161 @@ func (h *Handler) startListening() {
 	for {
 		msg := <-h.msg
 		go func() {
+			msgBytes, err := h.msgToJSON(msg)
+			if err != nil || len(msgBytes) == 0 {
```

Comment:
nit: I would assume `msgToJSON` would error out if `msgBytes` is 0 length.

Reply:
good point. removed.